src/core_plugins/metrics/public/components/vis_editor_visualization.test.js

```diff
@@ -0,0 +1,60 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+jest.mock('ui/visualize/loader/visualize_loader', () => ({}));
+
+import React from 'react';
+import { mount } from 'enzyme';
+import sinon from 'sinon';
+import VisEditorVisualization from './vis_editor_visualization';
+
+describe('getVisualizeLoader', () => {
+  let updateStub;
+
+  beforeEach(() => {
+    updateStub = sinon.stub();
```

Comment:
Just use `jest.fn()` instead, so we would not need any additional dependency on `sinon`. Also you can then replace the expect below to `expect(updateStub).toHaveBeenCalled()`